lib/pathresolver.js

```diff
@@ -39,6 +39,9 @@ PathResolver.prototype = {
       for (var j = 0, attr; j < constants.URL_ATTR.length; j++) {
         attr = constants.URL_ATTR[j];
         attrValue = dom5.getAttribute(node, attr);
+        if (attrValue === '') {
```

Comment:
Can you explain what is happening here a bit more? When is `attrValue` = `''`? If that is something that bundler is causing, Isn't that the real bug that needs to be fixed?